### PR TITLE
Creating Review component

### PR DESCRIPTION
**Pull request by sallamTanna**
Create a "Review" component, that holds a summary of the questions and their answers. #24

### CODE REVIEW

File: client/src/components/Review/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```
Comment (AbdallahAmmar96) [inline client/src/components/Review/index.js]: delete the semicolon

File: client/src/components/Review/style.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
+.review {
+  display: flex;
```
Comment (AbdallahAmmar96) [inline client/src/components/Review/style.css]: This style is useless 
The div tag is block

File: client/src/components/Review/style.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+.review-question {
+  color: #000000;
+  font-size: 1em;
+  font-weight: bold;
+}
+
+.review-answer {
+  color: #666666;
+  font-size: .9em;
```
Comment (AbdallahAmmar96) [inline client/src/components/Review/style.css]: This is too big for the answer

File: client/src/components/Review/style.css [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  flex-direction: column;
+}
+
+.review-guidelines {
+  text-align: right;
+  font-size: .5em;
+}
+
+.review-guidelines-href {
+  color: #E60085
```
Comment (AbdallahAmmar96) [inline client/src/components/Review/style.css]: remove the line under the word

File: client/src/components/Review/index.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+import './style.css'
+
+export default function Review(props) {
+  const { href, answer1, answer2, answer3 } = props;
+  return (
+    <container className='review'>
+      <div className='review-guidelines'><h2>Refer to <strong><a href={href} className='review-guidelines-href'>application guidelines</a></strong></h2></div>
```
Comment (AbdallahAmmar96) [inline client/src/components/Review/index.js]: split the tags 
indentation

File: client/src/components/Review/index.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import './style.css'
+
+export default function Review(props) {
+  const { href, answer1, answer2, answer3 } = props;
+  return (
+    <container className='review'>
```
Comment (AbdallahAmmar96) [inline client/src/components/Review/index.js]: There is no tag name container

File: client/src/components/Review/index.js (hunk 1 above)
Comment (sallamTanna) [reply client/src/components/Review/index.js]: Done

File: client/src/components/Review/style.css (hunk 2 above)
Comment (sallamTanna) [reply client/src/components/Review/style.css]: Done

File: client/src/components/Review/style.css (hunk 3 above)
Comment (sallamTanna) [reply client/src/components/Review/style.css]: Done

File: client/src/components/Review/style.css (hunk 4 above)
Comment (sallamTanna) [reply client/src/components/Review/style.css]: Done

File: client/src/components/Review/index.js (hunk 5 above)
Comment (sallamTanna) [reply client/src/components/Review/index.js]: Done

File: client/src/components/Review/index.js (hunk 6 above)
Comment (sallamTanna) [reply client/src/components/Review/index.js]: Edited

**Pull request merged**